apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -223,6 +240,16 @@ private async Task<int> ExecuteDmlAsync(CancellationToken cancellationToken)
                 return checked((int) count);
             }
 
+            private async Task<SpannerDataReader> ExecuteDmlReaderAsync(CancellationToken cancellationToken)
+            {
+                await Connection.EnsureIsOpenAsync(cancellationToken).ConfigureAwait(false);
+                var transaction = Transaction ?? Connection.AmbientTransaction ?? new EphemeralTransaction(Connection, Priority);
+                ExecuteSqlRequest request = GetExecuteSqlRequest();
+                var resultSet = await transaction.ExecuteDmlReaderAsync(request, cancellationToken, CommandTimeout).ConfigureAwait(false);
+                var enableGetSchemaTable = Connection.Builder.EnableGetSchemaTable;
+                return new SpannerDataReader(Connection.Logger, resultSet, Transaction?.ReadTimestamp, null, ConversionOptions, enableGetSchemaTable, CommandTimeout);
```

Comment:
We should support CommandBehaviour.CloseConnection here. I don't see whe we couldn't.